### ISSUE TITLE
custom babylon parser support

### ISSUE THREAD

**Issue by willin** [babel/babel-eslint#514]
.babelrc

```js
{
  "parserOpts": {
    "parser": "custom-fork-of-babylon-on-npm-here"
  }
}
```

if i use a custom babylon parser in babel, is there a similar option that i can parse eslint too?

```js
module.exports = {
  root: true,
  parser: 'babel-eslint',
  parseOptions: {
    // how to pass in "custom-fork-of-babylon-on-npm-here" 
  }
};
```

**Comment by hzoo** [comment]
you would have to replace `parser: babel-eslint` with something else

**Comment by hzoo** [comment]
This should work ^, and I guess you can read from babelrc now which might help (#711)
